experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -799,6 +799,69 @@ describe("SharedTree", () => {
 			unsubscribe2();
 		});
 
+		/**
+		 * the collab window includes all sequenced edits after the minimum sequence number
+		 * these tests test that undoing edits behind (i.e., with a seq# < to) the minimum sequence number works
+		 */
+		it("out of collab window", () => {
+			const provider = new TestTreeProviderLite(2);
+			const content = {
+				schema: stringSequenceRootSchema,
+				allowedSchemaModifications: AllowedUpdateType.None,
+				initialTree: ["A", "B", "C", "D"],
+			} satisfies InitializeAndSchematizeConfiguration;
+			const tree1 = provider.trees[0].schematizeInternal(content);
+
+			const { undoStack, redoStack, unsubscribe } = createTestUndoRedoStacks(
+				tree1.checkout.events,
+			);
+
+			provider.processMessages();
+			const tree2 = provider.trees[1].schematizeInternal(content);
+
+			const root1 = tree1.editableTree;
+			const root2 = tree2.editableTree;
+
+			// insert in first tree
```

Comment:
```suggestion
			// remove in first tree
```
Or just remove the comment. That line of code is pretty self-explanatory.